src/haxelib/client/FsUtils.hx

```diff
@@ -137,4 +137,33 @@ class FsUtils {
         try FileSystem.fullPath(path) catch (error:String) if (error == "std@file_full_path") errors++;
         return errors == 2;
     }
+
+	public static function getHomePath():String {
+		var home:String = null;
+		if (IS_WINDOWS) {
+			home = Sys.getEnv("USERPROFILE");
+			if (home == null) {
+				var drive = Sys.getEnv("HOMEDRIVE");
+				var path = Sys.getEnv("HOMEPATH");
+				if (drive != null && path != null)
+					home = drive + path;
+			}
+			if (home == null)
+				throw "Could not determine home path. Please ensure that USERPROFILE or HOMEDRIVE+HOMEPATH environment variables are set.";
+		} else {
+			home = Sys.getEnv("HOME");
+			if (home == null)
+				throw "Could not determine home path. Please ensure that HOME environment variable is set.";
+		}
+		return home;
+	}
+
+	// haxe 3.1.3 doesn't have FileSystem.absolutePath()
+	public static function absolutePath(path:String) {
+		if (StringTools.startsWith(path, '/') || path.charAt(1) == ':' || StringTools.startsWith(path, '\\\\')) {
```

Comment:
There's `haxe.io.Path.isAbsolute` method in std lib.

Reply:
I just copied and pasted this method as it was originally in `haxelib.client.Main`.

`haxe.io.Path.isAbsolute` wasn't in 3.1.3 either. I assumed that we're still keeping compatibility with that version?

Reply:
Removed the redundant method altogether.